model/pdata/generated_log.go

```diff
@@ -401,16 +401,16 @@ func (ms InstrumentationLibraryLogs) SetSchemaUrl(v string) {
 	(*ms.orig).SchemaUrl = v
 }
 
-// Logs returns the Logs associated with this InstrumentationLibraryLogs.
-func (ms InstrumentationLibraryLogs) Logs() LogSlice {
-	return newLogSlice(&(*ms.orig).Logs)
+// LogRecords returns the LogRecords associated with this InstrumentationLibraryLogs.
+func (ms InstrumentationLibraryLogs) LogRecords() LogSlice {
```

Comment:
Would be good to rename `LogSlice` to `LogRecordSlice`. If you do that add in pdata/logs.go:

```golang
// Deprecated: use LogRecordSlice
type LogSlice = LogRecordSlice
```